src/lib.rs

```diff
@@ -0,0 +1,201 @@
+/* Copyright (c) Fortanix, Inc.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! This crate provides an implementation of Base64 encoding/decoding that is
```

Comment:
Please review the documentation

Reply:
Looks good. It might be worth mentioning that (of course) this cannot hide the length of the data being processed. Which would be obvious to anyone who knows about ct programming but stating it is "independent of the data being processed" might lead to confusion on this point.

Reply:
This is already covered on line 28